test/run-verify-host-user.sh

```diff
@@ -53,63 +53,58 @@ fi
 
 export TEST_ALLOW_JOURNAL_MESSAGES
 
-# select tests
+# We only have one VM and tests should take at most one hour. So select those tests which exercise external API
+# (and thus are useful for reverse dependency testing and gating), and leave those which test cockpit-internal
+# functionality to upstream CI. We also need to leave out some which make too strict assumptions about the testbed.
 TESTS=""
 EXCLUDES=""
 RC=0
 if [ -n "$test_optional" ]; then
+    # don't include TestUpdates.testBasic: Testing Farm machines often have pending restarts/reboot
     TESTS="$TESTS
-         TestUpdates
-         TestAutoUpdates
-         TestStorage"
+         TestUpdates.testUpdateError
+         TestUpdates.testSecurityOnly
+         TestUpdates.testKpatch
+
+         TestAutoUpdates.testWithAvailableUpdates
+
+         TestStorage
+         "
 fi
 
 if [ -n "$test_basic" ]; then
-    # Testing Farm machines often have pending restarts/reboot
-    EXCLUDES="$EXCLUDES TestUpdates.testBasic TestUpdates.testFailServiceRestart"
+    TESTS="$TESTS
+        TestAccounts.testBasic
+        TestAccounts.testAccountLogs
 
-    # PCI devices list is not predictable
-    EXCLUDES="$EXCLUDES TestSystemInfo.testHardwareInfo"
+        TestJournal.testLogLevel
 
-    # No ABRT in CentOS/RHEL, thus not a test dependency
-    EXCLUDES="$EXCLUDES
-              TestJournal.testAbrtDelete
-              TestJournal.testAbrtReportCancel
-              TestJournal.testAbrtReport
-              TestJournal.testAbrtReportNoReportd
-              TestJournal.testAbrtSegv"
+        TestLogin.testBasic
+        TestLogin.testClientCertAuthentication
+        TestLogin.testSELinuxRestrictedUser
+        TestLogin.testTally
 
-    # FIXME: Often times out on at least c8s and f34 in TF
-    EXCLUDES="$EXCLUDES TestPages.testHistory"
+        TestServices.testBasic
+        TestServices.testBasicSession
 
-    TESTS="$TESTS
-        TestAccounts
-        TestBonding
+        TestSOS.testBasic
+
+        TestSystemInfo.testCPUSecurityMitigationsDetect
+
+        TestNetworkingBasic.testBasic
+        TestNetworkingVLAN
+        TestBonding.testBasic
         TestBridge
-        TestFirewall
-        TestKdump
-        TestJournal
-        TestLogin
-        TestNetworking
-        TestPackages
-        TestPages
-        TestServices
-        TestSOS
-        TestSystemInfo
         TestTeam
-        TestTerminal
-        TestTuned
-        "
+        TestNetworkingVLAN
 
-    # HACK: check-sos fails 100% on Testing Farm Fedora 36 without any error message; works in local tmt VM
-    if [ "$TEST_OS" = "fedora-36" ]; then
-        TESTS="${TESTS/TestSOS/}"
-    fi
```

Comment:
I was hoping that this would be fixed by the recent check-sos PR and the naughty, but it [still fails](https://artifacts.dev.testing-farm.io/a10eee44-0474-42d5-a3f3-834eda9c1276/) on F36. Back to draft.